packages/pages/src/components/hoursStatus/hoursStatus.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from "react";
+import c from "classnames";
+import { HoursType } from "../hours/types.js";
+import {
+  HoursIntervalManipulator,
+  HoursManipulator,
+} from "../hours/hoursManipulator.js";
+
+export interface StatusParams {
+  isOpen: boolean;
+  currentInterval: HoursIntervalManipulator | null;
+  futureInterval: HoursIntervalManipulator | null;
+  timeOptions?: Intl.DateTimeFormatOptions;
+  dayOptions?: Intl.DateTimeFormatOptions;
+}
+
+export interface TemplateParams {
+  currentTemplate?: (s: StatusParams) => React.ReactNode;
```

Comment:
One thing that could be a nice convenience is having these be able to take raw strings for cases where you don't care about the StatusParams and always want the same thing